doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_push.rst

```diff
@@ -0,0 +1,110 @@
+.. _ug_nrf54h20_suit_push:
+
+How to push SUIT payloads to multiple partitions
+################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+In the Software Updates for Internet of Things (SUIT), it is possible to push certain payloads separately from the SUIT envelope.
+The envelope can than find them in the device and use them as part of the firmware upgrade process.
+
+In Nordic Semiconductor's implementation, this functionality is provide by a mechanism called the DFU cache partitions.
+
+This guide explains how the DFU cache partitions work, and how to configure the build system and SUIT manifests to use them for pushing detached payloads.
+
+
+Reasons to push images separately from the SUIT envelope
+********************************************************
+
+Pushing images separately from the SUIT envelope can be useful in the following scenarios:
+
+* Pushing part of the image to an external memory chip.
+
+* Pushing parts of the update package to non-adjacent memory areas, separated by other data.
+
+* Reducing the amount of data that needs to be resent in case of communication failure to continue the update.
+
+* Pushing parts of the image through different communication channels.
+
+DFU Cache Partitions
+********************
+
+The payloads pushed to the device must be placed in so called “DFU cache partitions”, numbered 0..n.
+The DFU cache partitions hold data in the format of a CBOR map, in which the keys are URI strings and the values are binary images.
+
+When the Secure Domain processes the manifest and encounters a “fetch” directive it first checks if a given URI is present in its “integrated payloads” section.
```

Comment:
```suggestion
When the Secure Domain processes the manifest and encounters a fetch directive, it first checks if a given URI is present in its “integrated payloads” section.
```

Reply:
I've changed to ``suit-directive-fetch``, I think it will be more understandable